team-config/aat/all.yaml

```diff
@@ -133,6 +133,10 @@ gateways:
         component: user-profile-api
       - product: rd
         component: profile-sync
+      - product: rd
+        component: judicial-data-load
```

Comment:
Is this an API ? Doesn't sound so by name.

Reply:
its kind of batch process